lib/src/impl/realtime/realtime.dart

```diff
@@ -29,15 +29,15 @@ class Realtime extends PlatformObject
 
   @override
   Future<int> createPlatformInstance() async {
-    var handle = await invokeRaw<int>(
+    final handle = await invokeRaw<int>(
         PlatformMethod.createRealtimeWithOptions, AblyMessage(options));
     _realtimeInstances[handle] = this;
     return handle;
   }
 
   // The _connection instance keeps a reference to this platform object.
```

Comment:
Should this be a `TODO(tiholic) ...` to make it easier to spot that this requires more work,
or alternatively a GitHub issue

Reply:
TODO removed and `UnimplementedError` is thrown